### PR TITLE
Deprecate update api config endpoints

### PR DESCRIPTION
**Pull request by davidjiglesias**
Hello team,

This PR updates the deprecation of update_api_config endpoints and also adds the remote commands configuration.

Best regards,

David J. Iglesias

### CODE REVIEW

File: source/user-manual/api/configuration.rst [hunk 1]
```diff
@@ -256,13 +246,23 @@ access
...
+| Sub-fields | Allowed values       | Default value | Description                                                                                                                                                                                                                                                                                                                                 |
++============+======================+===============+=============================================================================================================================================================================================================================================================================================================================================+
+| enabled    | yes, true, no, false | true          | Enable or disable uploading configurations with remote commands through the Wazuh API. When this option is disabled it is not possible to upload **ossec.conf** files with the  :ref:`<command> <command>` option inside the :ref:`localfile tag <reference_ossec_localfile>` as well as the :ref:`wodle "command" option <wodle_command>`. |
```
Comment (s-ocando) [inline source/user-manual/api/configuration.rst:261]: WARNING: undefined label: command (if the link has no caption the label must precede a section header)

File: source/user-manual/api/configuration.rst [hunk 2]
```diff
@@ -54,9 +54,18 @@ Here are all the available settings for the ``api.yaml`` configuration file. For
...
+           exceptions: []
+
+       wodle_command:
+          enabled: yes
+          exceptions: []
+
 .. warning::
 
-    If running a cluster, the master will NOT send its local Wazuh API configuration file to the workers. Each node provides its own Wazuh API. If the configuration file is changed in the master node, the user should manually update the workers Wazuh API configuration in order to use the same one. Take care of not overwriting the IP and port in the local configuration of each worker. The Wazuh API endpoint :api-ref:`PUT /cluster/api/config <operation/api.controllers.cluster_controller.put_api_config>` can be used to change any or all of the Wazuh API configuration files in the cluster nodes.
+    If running a cluster, the master will NOT send its local Wazuh API configuration file to the workers. Each node provides its own Wazuh API. If the configuration file is changed in the master node, the user should manually update the workers Wazuh API configuration in order to use the same one. Take care of not overwriting the IP and port in the local configuration of each worker.
```
Comment (s-ocando) [inline source/user-manual/api/configuration.rst:68]: I would suggest changing "Take care of not overwriting" to "Be sure to not overwrite"

File: source/user-manual/api/configuration.rst (hunk 1 above)
Comment (vicferpoy) [reply source/user-manual/api/configuration.rst]: Done.

File: source/user-manual/api/configuration.rst (hunk 2 above)
Comment (vicferpoy) [reply source/user-manual/api/configuration.rst]: Done.

**Review by s-ocando — APPROVED** [review]
LGTM

**Review by s-ocando — APPROVED** [review]
LGTM

**Pull request merged**